### PR TITLE
UX: add maximum image size in the instructions

### PR DESCRIPTION
**Pull request by saiqulhaq**

### CODE REVIEW

File: app/assets/javascripts/discourse/components/learn-more.js.es6 [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export default Ember.Component.extend({
+  type: null,
+
+  uploadInfo(type){
+    $.ajax({
```
Comment (ZogStriP) [inline app/assets/javascripts/discourse/components/learn-more.js.es6]: We wrapped jQuery's ajax call in `Discourse.ajax` which is now the recommended way of talking to the API

File: app/assets/javascripts/discourse/components/learn-more.js.es6 [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    }});
+  },
+
+  actions: {
+    show(){
+      switch(this.get('type')){
+        case 'max_image_size':
+          this.uploadInfo(this.get('type'));
+          break;
+        default:
```
Comment (ZogStriP) [inline app/assets/javascripts/discourse/components/learn-more.js.es6]: Not sure we need that.

**Comment by ZogStriP** [comment]
Also, since it's mostly a UI change, can we get before/after screenshots?

File: app/assets/javascripts/discourse/components/learn-more.js.es6 (hunk 2 above)
Comment (saiqulhaq) [inline app/assets/javascripts/discourse/components/learn-more.js.es6]: Oh yea I will remove this, it was for debugging, thanks

File: app/assets/javascripts/discourse/components/learn-more.js.es6 (hunk 1 above)
Comment (saiqulhaq) [inline app/assets/javascripts/discourse/components/learn-more.js.es6]: OK I will update it

**Comment by saiqulhaq** [comment]
@ZogStriP Sure I will upload it soon

**Comment by saiqulhaq** [comment]
Hi @ZogStriP 
please check following screenshot

### Before
screen shot 2016-04-11 at 4 15 48 pm

### After
screen shot 2016-04-11 at 4 16 12 pm
screen shot 2016-04-11 at 4 18 51 pm

Thanks

**Comment by coding-horror** [comment]
This seems like a lot of effort to present something that I am not sure is very useful to many users?

**Comment by saiqulhaq** [comment]
Hi
please check following [topic](https://meta.discourse.org/t/show-image-file-size-limit-on-user-preference-page/41707)

Actually I found some challenge while deciding the UI, maybe you have another idea?
It's OK if you think it's not too useful, I will have no hard feeling for that :)

**Comment by coding-horror** [comment]
OK we will take discussion to that topic.. I very much appreciate the effort, I just feel this is not the correct fix, and adds a lot of changes and translations..

**Comment by saiqulhaq** [comment]
OK great, thanks

**Pull request closed without merging**